### PR TITLE
5.6.8 Release Notes

### PR DESCRIPTION
**Pull request by Chr1st0ph3rTurn3r**

### CODE REVIEW

File: docs/release_notes_128t_5.6.md [hunk 1]
```diff
@@ -28,6 +28,123 @@ Before upgrading please review the [**Upgrade Considerations**](intro_upgrade_co
...
+------
+- **I95-50286 Rebooting a node of an HA pair from Linux breaks routing:** Resolved an issue where a delay in the shutdown process caused a node to take over a VRRP interface, creating routing issues. 
+------
+- **I95-50331 System fails to synchronize keys on startup:** The SSR now dynamically updates the `rsync IP host address` from the non forwarding HA sync interfaces, and will fall back to the `global.init` host IPs if they don't exist.
+------
+- **I95-50363 MOS Metrics not refreshing:** Resolved an issue where the SLA and MOS values were not being updated in the stats (or PeerPathTable) when a BFD session was brought down. The SLA and MOS stats are now set to 0 when the BFD session is brought down.
+------
+- **I95-50376 Failure to make config changes after rollback:** Resolved an issue where commits would not take effect after rolling back an HA router, because of older/newer version conflicts. 
+------
+- **I95-50445 i40e and ice devices enter malicious descriptor detection state, preventing forwarding of traffic:** Resolved an issue where fragmented packet chains larger than 8 buffers were discarded causing a malicious descriptor event.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_5.6.md:110]: This is a serious issue and should have a log snippet so customers and SEs can be made more knowlegable of the issue.
the following is an example of a log messages see in `dpdk.log` indicating the occurrence of the issue:
```
[DPDK| -- ] ERROR (00007f03ec18e700) i40e_dev_alarm_handler(): ICR0: malicious programming detected
[DPDK| -- ] WARN  (00007f03ec18e700) i40e_handle_mdd_event(): Malicious Driver Detection event 0x02 on TX queue 6 PF number 0x01 VF number 0x00 device 0000:08:00.1
[DPDK| -- ] WARN  (00007f03ec18e700) i40e_handle_mdd_event(): TX driver issue detected on PF
```

Comment (MichaelBaj) [reply docs/release_notes_128t_5.6.md:110]: Also, I do know that I didn't make this comment in 6.1.2 release notes, but I should have.

File: docs/release_notes_128t_5.6.md [hunk 2]
```diff
@@ -28,6 +28,123 @@ Before upgrading please review the [**Upgrade Considerations**](intro_upgrade_co
...
+------
+- **I95-51006 Nodes stuck in connected state after upgrade:** On an HA conductor, if the user is performing an upgrade on the first conductor node and that user makes a config commit during the upgrade, then the configuration's modified time will become out of sync between the two conductor nodes. When the conductor first node is finished upgrading the result is a loop where the configuration keeps getting committed by each node back and forth until a new commit is made. This issue has been resolved by allowing the peer conductor node to accept the config despite the perceived version disparity. Please note performing a commit mid upgrade is not supported.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_5.6.md:136]: you have this issue listed twice.

**Review by MichaelBaj — CHANGES_REQUESTED** [review]

File: docs/about_releases.mdx [hunk 3]
```diff
@@ -25,7 +25,7 @@ Version `6.1.0` introduces changes to the SSR software release model. Every six
 | Version | Initial GA Version | First Release Shipping Date | Latest GA Version | End of Software Engineering support | End of Support |
 | --- | --- | --- | --- | --- | --- |
 | Release 6.1 | [6.1.0](release_notes_128t_6.1.md#release-610-55) | April 14, 2023 | [6.1.2](release_notes_128t_6.1.md#release-612-7) | January 14, 2024 |   July 14, 2024 |
-| Release 5.6 | [5.6.7](release_notes_128t_5.6.md#release-567-4) | March 16, 2023 | [5.6.8](release_notes_128t_5.6.md#release-568-x) | June 16, 2024 | December 16, 2024 |
+| Release 5.6 | [5.6.7](release_notes_128t_5.6.md#release-567-4) | March 16, 2023 | [5.6.8](release_notes_128t_5.6.md#release-568-8) | June 16, 2024 | December 16, 2024 |
```
Comment (MichaelBaj) [inline docs/about_releases.mdx:28]: This should be 5.6.8 in the link

File: docs/release_notes_128t_5.6.md [hunk 4]
```diff
@@ -93,11 +93,11 @@ and there are established flows for any of these services, a link flap triggerin
...
+- **I95-50247 Duplicate peer path alarms:** Resolved an issue where both BFD and the path MTU feature were generating alarms for the same peer path being down. The criteria for which peer path state changes can trigger peer path events has been tightened.
 ------
 - **I95-50262 Routers disconnected from their conductor may have incorrect log rotation settings:** Resolved an issue where a managed router was not able to pull down the configuration from the Conductor - which includes the log rotation config. The default salt log rotation configuration has been improved, preventing the log from growing too large before the connection to the Conductor can be established. 
 ------
-- **I95-50269 Router clone operation fails:** Implemented checks to prevent cloning obsolete elements and internal lists/containers on legacy versions of the SSR software (pre-4.4).
+- **I95-50269 Router clone operation fails:** Implemented checks to prevent cloning obsolete elements and internal lists/containers on legacy versions of the SSR software (SSR Software </= version 4.4).
```
Comment (MichaelBaj) [inline docs/release_notes_128t_5.6.md:100]: I'm not familiar with </= is that a way to escape an equal sign?

**Review by MichaelBaj — DISMISSED** [review]

**Review by MichaelBaj — APPROVED** [review]

**Review by MichaelBaj — DISMISSED** [review]

File: docs/release_notes_128t_5.6.md [hunk 5]
```diff
@@ -43,18 +43,16 @@ The Juniper SSR team does not publicly disclose known or resolved CVEs in our pu
 	- Enforce setting of packet location.
 	- Add the ability to walk packet buffer pools, count the locations, and display.
 ------
-- **I95-51169 Buffer tracking improvements:** The following improvements have been made to Buffer Tracking:
+- **I95-51169, I95-51173 Buffer tracking improvements:** The following improvements have been made to Buffer Tracking:
 	- Refined packet buffer location tracking to better identify buffers in use for `TSI` collection.
 	- Added new metrics for tracking utilization of packet pools.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_5.6.md:48]: Please add paths for the new metrics

**Review by MichaelBaj — CHANGES_REQUESTED** [review]

File: docs/release_notes_128t_5.6.md [hunk 6]
```diff
@@ -45,8 +45,12 @@ The Juniper SSR team does not publicly disclose known or resolved CVEs in our pu
 ------
 - **I95-51169, I95-51173 Buffer tracking improvements:** The following improvements have been made to Buffer Tracking:
 	- Refined packet buffer location tracking to better identify buffers in use for `TSI` collection.
-	- Added new metrics for tracking utilization of packet pools.
 	- Provide more diagnostic information, when possible.
+	- Added the following new metrics for tracking utilization of packet pools:
+		- `FastLanePacketPool`
```
Comment (MichaelBaj) [inline docs/release_notes_128t_5.6.md:50]: Sorry, I should have been more clear. We need the full path.
e.g., `packet-processing/classifier/received/ipv4-fragmented`
Not sure what the equivalent is for these metrics. That path would be used for both the GUI as well as PCLI.

**Review by MichaelBaj — CHANGES_REQUESTED** [review]

**Review by MichaelBaj — APPROVED** [review]

**Review by jeffmart-jnpr — APPROVED** [review]

**Pull request merged**